2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2047 @@
+48. School of Computing, Communication and Business, Hochschule für Technik und Wirtschaft Berlin, Germany
+- * axel.loewe@kit.edu, gunnar.seemann@universitaets-herzzentrum.de
+- † Contributed equally
+bibliography: 'references.bib'
+title: |
+    An Environment for Sustainable Research Software in Germany and Beyond:
+    Current State, Open Challenges, and Call for Action
+---
+
+Sustainable Software Development; Academic Software; Software
+Infrastructure; Software Training; Software Licensing; Research Software
+
+Background
+==========
+
+> Meet Kim, who is currently a post-grad PhD student in researchonomy at
+> the University of Arcadia (UofA). We will follow Kim’s fictional career in
+> order to understand different aspects of research software
+> sustainability. Note that in Kim’s world, many of the changes this
+> paper calls for have already been implemented.  
+> (In our example, Kim is a female person. Of course, research software
+> engineers (RSEs) can be any gender.)
+
+Computational analysis of huge data sets, computer-based simulations,
+and software technology play a central role for virtually all scientific
+breakthroughs of the 21st century. The first “picture” of a black hole
+may be the most prominent recent example where astrophysical experiments
+and the collection and processing of data had to be complemented with
+sophisticated algorithms and software to enable research excellence (The
+Event Horizon Telescope Collaboration et al. 2019). Similarly, it is
+research software that allows us to get a glimpse of the consequences
+our actions today have on the climate of tomorrow. However, an
+implication of computer-based research is that findings and data can
+only be reproduced, understood, and validated if the software that was
+used in the research process is available and
+maintained.  
+At the same time, sustaining research software, and in particular open
+research software, comes with a number of challenges. Commercial
+research software often has revenue flows that can facilitate
+sustainable software development, maintenance, and documentation as well
+as the operation of adequate infrastructure. A large share of researchers (want to or have to) 
+base their research on software that is often developed in-house or as a community
+effort. Many of these software stacks can not be
+sustained – often because research software was not a first class
+deliverable in a research project and hence remained in a prototype
+state, or because of missing incentives and resources to maintain the
+software after project funding ended. Another fundamental difference to
+industrial software development is that most developers of academic
+research software (often doctoral students or postdoctoral researchers)
+never receive training in sustainable software development or consider themselves professional software developers (cite survey here). In
```

Comment:
```suggestion
never receive training in sustainable software development (cite survey here). In
```

Does self-awareness really play a role here? I don't think so (professional devs may or may not consider themselves professional, but it doesn't change the professionality of their work).